docker-compose.yml

```diff
@@ -0,0 +1,26 @@
+version: '3.8'
+
+services:
+  audio-format-service:
+    build:
+      context: .
+      dockerfile: audio-format-service/Dockerfile
+    ports:
+      - "8001:8001"
+    environment:
+      - PYTHONPATH=/app/src
+      - TEMP_DIR=/app/temp
+    volumes:
+      - ./data:/app/data
+      - ./models:/app/models
+    healthcheck:
+      test: ["CMD", "curl", "-f", "http://localhost:8001/health"]
+      interval: 30s
+      timeout: 10s
+      retries: 3
+      start_period: 40s
+    restart: unless-stopped
+
+volumes:
+  data:
+  models:
```

Comment:
Kiro comment: 🤔 These named volumes are declared but service uses bind mounts (`./data` and `./models`). Two options:
1. Remove volume declarations (if we only need bind mounts for development)
2. Use named volumes in production, bind mounts in dev (docker-compose.override.yml)

For this project, bind mounts seem intentional for easy access. Recommend removing unused volume declarations in PR #17.